resim/transforms/python/se3_python.cc

```diff
@@ -0,0 +1,28 @@
+// Copyright 2023 ReSim, Inc.
+//
+// Use of this source code is governed by an MIT-style
+// license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+#include <pybind11/eigen.h>
+#include <pybind11/pybind11.h>
+
+#include <utility>
+
+#include "resim/transforms/se3.hh"
+
+namespace resim::transforms {
+namespace py = pybind11;
+
+// A simple pybinding of SE3
+// TODO(mikebauer) Add frames and other operators
+PYBIND11_MODULE(se3_python, m) {
+  py::class_<SE3>(m, "SE3")
+      .def_readonly_static("DIMS", &SE3::DIMS)
+      .def_readonly_static("DOF", &SE3::DOF)
+      .def(py::init<>())
+      .def("log", &SE3::log)
+      .def("exp", [](const SE3::TangentVector &arg) { return SE3::exp(arg); });
```

Comment:
I think I'm of the opinion that this class has to at least be able to expose (read-only) copies of the translation and rotation matrix. This probably requires a PyBind for SO3 too, but I think that's worth it.

Reply:
I've significantly increased the interface here. Let me know if you like the way it is now.